src/electron/process_manager.ts

```diff
@@ -404,13 +396,33 @@ class Tun2socks extends ChildProcessHelper {
     args.push('--netif-ipaddr', TUN2SOCKS_VIRTUAL_ROUTER_IP);
     args.push('--netif-netmask', TUN2SOCKS_VIRTUAL_ROUTER_NETMASK);
     args.push('--socks-server-addr', `${this.proxyAddress}:${this.proxyPort}`);
-    args.push('--loglevel', this.isInDebugMode ? 'debug' : 'error');
     args.push('--transparent-dns');
     if (isUdpEnabled) {
       args.push('--socks5-udp');
       args.push('--udp-relay-addr', `${this.proxyAddress}:${this.proxyPort}`);
     }
+    args.push('--loglevel', this.isInDebugMode ? 'debug' : 'error');
+    if (this.isInDebugMode) {
+      // BReactor is extremely verbose
+      args.push('--channel-loglevel', 'BReactor', 'info');
+    }
 
     this.launch(args);
   }
 }
+
+function logExit(processName: string, exitCode?: number, signal?: string) {
+  const prefix = `[EXIT - ${processName}]: `;
+  const exitReason = exitCode ?? signal;
```

Comment:
Remove `exitReason`. It's not needed.
This would be clearer:
```js
if (exitCode !== null) {

} else if if (signal !== null) {

}
```

Reply:
`exitReason` covers both null and undefined

Reply:
It's harder to ensure it's correct.
What if signal == exitCode?
What if exitCode == null and signal == 0?

Let's be more explicit and test for null, which aligns well with the API documentation.

Reply:
Done in 8268a9684b86840a8c0b77b436e0fc897addf7c4

Reply:
Done in 92b15d7408ec15d5112140945ecf0bf5c8b57bb5